miranda/convert/data/ecmwf_cf_attrs.json

```
@@ -163,20 +159,16 @@
       "units": "kg m-3"
     },
     "sd": {
-      "_cf_variable_name": "snw",
-      "_corrected_units": {
-        "era5-land": "Mg m-2",
-        "era5-single-levels": "Mg m-2",
-        "era5-single-levels-preliminary-back-extension": "Mg m-2"
-      },
+      "_cf_variable_name": "swe",
+      "_corrected_units": "m",
       "_invert_sign": false,
       "_offset_time": false,
       "_transformation": false,
       "cell_methods": "time: point",
-      "description": "Snow thickness in m of liquid water equivalent converted to snow amount using a water density of 1000 kg/m³.",
-      "long_name": "Surface snow amount",
-      "standard_name": "surface_snow_amount",
-      "units": "kg m-2"
+      "description": "This parameter is the depth of snow from the snow-covered area of a grid box. Its units are metres of water equivalent, so it is the depth the water would have if the snow melted and was spread evenly over the whole grid box. The ECMWF Integrated Forecast System represents snow as a single additional layer over the uppermost soil level. The snow may cover all or part of the grid box.",
```

```suggestion
      "description": "This parameter is the average amount of snow of a grid box. Its units are metres of water equivalent, so it is the depth the water would have if the snow melted and was spread evenly over the whole grid box. The ECMWF Integrated Forecast System represents snow as a single additional layer over the uppermost soil level. The snow may cover all or part of the grid box.",
```
I think this makes the first sentence more coherent with the second one, but I may be misunderstanding?

---

This is taken directly from ECMWF, haha

---

I am in agreement, though. Your changes sound better. Merging.

---

haha wow. I really do not like SWE because of this constant ambiguity with the word "depth"...